cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1190,6 +1193,12 @@ construct_runtime!(
 
 		AssetRewards: pallet_asset_rewards = 61,
 
+		// Bridges permissionless lanes.
+		XcmOverAssetHubRococo: pallet_xcm_bridge::<Instance1> = 62,
+		BridgeRococoMessages: pallet_bridge_messages::<Instance1> = 63,
+		BridgeRelayers: pallet_bridge_relayers::<Instance1> = 64,
+		ToRococoOverAssetHubRococoXcmRouter: pallet_xcm_bridge_router::<Instance1> = 65,
```

Comment:
```suggestion
		ToRococoOverAssetHubRococoXcmRouter: pallet_xcm_bridge_router::<Instance2> = 65,
```

InstanceX needs to be unique and above there is already `Instance1` used for `ToRococoXcmRouter: pallet_xcm_bridge_router::<Instance1> = 34,`, so let's change it here to Instance2 and it needs to be also Instance2 here:
https://github.com/paritytech/polkadot-sdk/pull/7831/files#diff-b5e74617755e07fd3b6df03572d63c56fa6690328a9ec0749f8dd8dd9ec9ad49R219
```
pub type ToRococoOverAssetHubRococoXcmRouterInstance = pallet_xcm_bridge_router::Instance2;
```